JS/Lesson34/goksel/recipe-app/src/modules/recipes/NewRecipe/NewRecipeForm/newRecipeHelper.ts

```diff
@@ -0,0 +1,29 @@
+import { Areas, Categories } from "../../models";
+
+export const handlestrIngredientsAndMeasures = (data?: string) => {
+    const ingredientsAndMeasures = data ? data.split(",") : undefined;
+    let cleanedData = {};
```

Comment:
If you type cleanedData like this:
`let cleanedData:{ [key: string]: string } = {};`
you can remove `// @ts-ignore` comments below